code/drasil-theory/lib/Theory/Drasil/DataDefinition.hs

```diff
@@ -87,7 +87,7 @@ instance HasDecRef          DataDefinition where getDecRefs = ddPkt pktDR
 -- | Equal if 'UID's are equal.
 instance Eq                 DataDefinition where a == b = (a ^. uid) == (b ^. uid)
 -- | Finds the derivation of the 'DataDefinition where'. May contain Nothing.
-instance HasDerivation      DataDefinition where derivations = ddPkt pktMD
+instance MayHaveDerivation      DataDefinition where derivations = ddPkt pktMD
```

Comment:
Minor code style issue: we align the type arguments of lists of instances.

Reply:
Do not understand what should I change, do you mean this? @balacij (I use the above example)
instance HasDerivation               DataDefinition where derivations = ddPkt pktMD
instance MayHaveDerivation      DataDefinition where derivations = ddPkt pktMD <-- now there argument types are aligned

Reply:
Just so you know, you can wrap code with 3 backticks to make a code block. Using the code block will use a monospaced font, so seeing the spacing is easier :smile: 

I _think_ that looks right. The point is that the `DataDefinition`s should be vertically aligned there.

Reply:
So what should I do is just aligning all ```DataDefinition```
Here is an example (as stated beyond):
```
instance HasDerivation               DataDefinition where derivations = ddPkt pktMD
instance MayHaveDerivation           DataDefinition where derivations = ddPkt pktMD
```
Now it is aligned

Reply:
Yes, but that looks like you added too much space. Align the `DataDefinition where...` onwards with the others listed in the files, please.

Reply:
Sorry for the delay. (Working on other things) In this modification, I changed three files: 
```code/drasil-theory/lib/Theory/Drasil/DataDefinition.hs```
```code/drasil-theory/lib/Theory/Drasil/GenDefn.hs```
```code/drasil-theory/lib/Theory/Drasil/InstanceModel.hs```

They have similar alignment problems, but now all these issues have been solved. As you can see: https://github.com/JacquesCarette/Drasil/pull/3653/files#diff-bab435096d39e524c93900a45bea3ad6e1b439001c0a54ea1f27d94d28bd824e